src/index.html

```diff
@@ -11,7 +11,172 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class="form-field">
+        <legend>Personal information</legend>
+        <label class="input">
+          Surname:
+          <input
+          type="text"
+          name="Surname"
+          required
+          autocomplete="off"
+          >
```

Comment:
```suggestion
          <input
            type="text"
            name="Surname"
            required
            autocomplete="off"
          >
```
Fix all cases
![image](https://user-images.githubusercontent.com/96409563/224264961-cd8e4091-e96d-4e7d-a1ed-55410a02688d.png)